django/conf/global_settings.py

```diff
@@ -194,6 +194,9 @@ def gettext_noop(s):
 # Port for sending email.
 EMAIL_PORT = 25
 
+# Whether to send SMTP date headers in the local time zone
```

Comment:
Is "SMTP date header" the correct term? I'm not sure this has anything to do with SMTP.